src/DynamoApplications/StartupUtils.cs

```diff
@@ -143,7 +143,7 @@ public static void PreloadShapeManager(ref string geometryFactoryPath, ref strin
 
             var versions = new[]
             {
-                    new Version(225,0,0),
+                    new Version(225,4,0),
```

Comment:
Was this just for testing or something?

Reply:
Just for testing, this will be reverted for final commit

Reply:
@QilongTang this is still set. Can you add a test to the preloader tests using the new logic?

Reply:
Adding a test